frontend/model/contracts/chatroom.js

```diff
@@ -219,12 +212,13 @@ sbp('chelonia/defineContract', {
 
         emitMessageEvent({ contractID, hash })
         setReadUntilWhileJoining({ contractID, hash, createdDate: meta.createdDate })
+
         if (username === loggedIn.username) {
           const { type, privacyLevel } = state.attributes
-          const isPrivateDM = type === CHATROOM_TYPES.INDIVIDUAL && privacyLevel === CHATROOM_PRIVACY_LEVEL.PRIVATE
-          if (isPrivateDM) {
+          const isDirectMessage = type === CHATROOM_TYPES.INDIVIDUAL && privacyLevel === CHATROOM_PRIVACY_LEVEL.PRIVATE
```

Comment:
Same here, it's a direct message if `type === CHATROOM_TYPES.DIRECT_MESSAGE`, no need to check the privacy level